docs/examples/ex_match.rst

```diff
@@ -0,0 +1,188 @@
+.. _ex_match:
+
+Pair Modelled and Observed Data
+===============================
+
+One common analytical need is to obtain a combined data set of modelled and
+observed observations at the same times and locations.  This can be done using
+the :py:func:`collect_inst_model_pairs` function in :ref:`utils-match`.  This
+routine takes a date range as input, and then extracts modelled observations at
+the specified instrument location.  However, it does not interpolate in time.
+Details about the interpolation of modelled data onto the instrument location
+can be found in the :ref:`utils-extract` routine,
+:py:func:`extract_modelled_observations`.
+
+
+.. _ex_match-loc:
+    
+Match Data by Location
+----------------------
+    
+In the example below, we load a DINEOFs file created for testing purposes and
+pair it with C/NOFS IVM data.  This is a global example for a single time,
+since in this instance DINEOFs was used to create a day-specific emperical
+moodel.  Comparisons with output from a global circulation model would look
+different, as one would be more likely to desire the the closest observations
+to the model time rather than all observations within the model time.
+
+This example uses the external modules:
+
+#. pysat
+#. pysatNASA
+
+Load a test model file from pyDINEOFs. It can be downloaded from the GitHub
+repository using the standard :py:meth:`pysat.Instrument.download` method. We
+will also set the model-data input keyword arguements that are determined by
+the model being used.  The DINEOFs test data applies for an entire day, so the
+:py:data:`pair_method` is ``'all'`` and the :py:data:`time_method` is ``'max'``.
+
+::
+
+
+   import datetime as dt
+   from os import path
+   import pysat
+   import pysatNASA
+   import pysatModels as ps_mod
+
+   # Set the instrument inputs
+   inst_module = ps_mod.models.pydineof_dineof
```

Comment:
I like the idea of using both, since neither option is necessarily better than the other.  We'll probably be annoying half of the users half of the time.